doc/source/data/examples/batch_training.ipynb

```diff
@@ -13,6 +13,16 @@
    "source": [
     "# Introduction\n",
     "\n",
+    "```{tip}\n",
+    "Ray provides multiple ways to implement this workload.\n",
+    "\n",
+    "* Ray Core will be the lowest level of abstraction and the most performant.\n",
```

Comment:
It's actually probably not accurate: high level libraries have many performance optimizations and ray core will be slower if users don't implement those optimizations.

It's more like ray core can be the fastest but it requires lots of work/tuning to get that benefit.

Reply:
I'll make the tip more specific to the simple workloads we are talking about here

Reply:
For the sample workloads, it may also not be true. Have we benchmarked them?

I would position Ray core as a last resort only if the high level libraries don't meet the requirement.

My concern with the current statements is that people will have the impression that high level libraries are slow and they will directly try try core.

Reply:
Yes, 100% agree with Jiajun here. The point we should make is that Core offers more _control_ over workloads. This is a two edged sword: you can tune it for performance, but it will probably be a lot more work to implement the necessary backpressure, batching, pipelining, and so on optimizations. For most users, libraries will work better out of the box.

Reply:
We have benchmarked them, Core is 5x - 10x faster. @richardliaw can provide details

Reply:
I think you're missing the point here: it's not about microbenchmarks, but about the use cases. This is kind of like saying assembly is faster and you should always use it for performance: technically correct, but most users won't be able to implement assembler faster than higher level APIs.